repository/snowflake.go

```diff
@@ -16,42 +16,44 @@ package repository
 
 import (
 	"context"
-	"database/sql"
 	"fmt"
+	"reflect"
 	"strings"
-	"time"
 
 	"github.com/huandu/go-sqlbuilder"
+	"github.com/jmoiron/sqlx"
```

Comment:
Because we are using method  MapScan from sqlx lib
```
// MapScan using this Rows.
func (r *Rows) MapScan(dest map[string]interface{}) error {
	return MapScan(r, dest)
}
```
For example on line https://github.com/conduitio-labs/conduit-connector-snowflake/blob/develop/source/iterator/snapshot.go#L142